trace.go

```diff
@@ -148,7 +148,8 @@ func (e *traceExporter) pushTraceSpans(ctx context.Context, node *commonpb.Node,
 	ctx, cancel := newContextWithTimeout(ctx, e.o.Timeout)
 	defer cancel()
 
-	return e.client.BatchWriteSpans(ctx, &req)
+	err := e.client.BatchWriteSpans(ctx, &req)
+	return 0, err
```

Comment:
Rhere will never be any dropped spans for stackdriver, is that correct?

Reply:
I think if `err` is not `nil`, we should return `len(protoSpans)` here.

Reply:
@dinooliva it will be all or none.

@songy23 fixed it.